### PR TITLE
Split the test cases for mapping width and height to aspect ratio

### PR DESCRIPTION
**Pull request by cathiechen**
Hi,
This patch tries to split up the test cases, to make sure that each element has its own test case, so that the test won't break in the middle of test cases and affect other images' test.
PTAL, thanks!

### CODE REVIEW

**Review by rwlbuis — APPROVED** [review]
LGTM.

File: html/rendering/replaced-elements/attributes-for-embedded-content-and-images/img-aspect-ratio.html [hunk 1]
```diff
@@ -31,57 +31,84 @@
   test_computed_style_aspect_ratio("input", {type: "submit", width: width, height: height}, "auto");
 }
 
+// ======The computed style test cluster======
```
Comment (emilio) [inline html/rendering/replaced-elements/attributes-for-embedded-content-and-images/img-aspect-ratio.html:34]: This comment doesn't seem super-useful.

**Review by emilio — APPROVED** [review]
looks fine to me, thanks!

**Comment by cathiechen** [comment]
Hi Rob and Emilio,
Thanks for the review!

**Pull request merged**